City-of-Helsinki/helsinki-paatokset
# PP-76: News feed integration module

Add news feed integration module, changes:

- New custom module paatokset_news_importer
- Added new node type imported_article
- Custom module contains migrations
- Module contains view for imported_articles feed, will be styled later
- Contains template for imported articles, which is basically a copy+paste of article template

To test:

- `make drush-cr; make drush-cim`
- paatokset_news_importer should ne installed now
- run new custom migrations: `PARTIAL_MIGRATE=1 drush migrate:import paatokset_news_images; PARTIAL_MIGRATE=1 drush migrate:import paatokset_news` - env variable set here prevents helfi_api_base event subscriber from failing the migration, but does not affect this one
- Content of type imported_article should be imported from the RSS feed
